applications/ConstitutiveLawsApplication/custom_constitutive/rule_of_mixtures_law.cpp

```diff
@@ -1201,35 +1257,76 @@ template<unsigned int TDim>
 void ParallelRuleOfMixturesLaw<TDim>::FinalizeMaterialResponsePK1(Parameters& rValues)
 {
     const Properties& r_material_properties = rValues.GetMaterialProperties();
-
+	// Get Values to compute the constitutive law:
+    Flags& r_flags = rValues.GetOptions();
+    // Previous flags saved
+    const bool flag_const_tensor = r_flags.Is(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR);
+    const bool flag_stress       = r_flags.Is(ConstitutiveLaw::COMPUTE_STRESS);
+    const bool flag_strain       = r_flags.Is(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN);
+	// All the strains must be the same, therefore we can just simply compute the strain in the first layer
```

Comment:
Tabs (affects all tabs)
```suggestion
    // All the strains must be the same, therefore we can just simply compute the strain in the first layer
```